addons/mail/static/src/core/web/chatter.js

```diff
@@ -199,13 +202,16 @@ export class Chatter extends Component {
      */
     get toFollowersText() {
         const threadName = this.state.thread.displayName || this.state.thread.name;
+        const followers = this.state.thread.followers.slice(-5).map(({ partner }) =>
+            sprintf('<span title="%(email)s">%(emailWithoutDomain)s</span>', {
+                email: escape(partner.email),
+                emailWithoutDomain: escape(partner.emailWithoutDomain),
+            })
+        );
         const toFollowersText = threadName
-            ? sprintf(
-                  _t(
-                      '<span class="fw-bold">To:</span> <span class="fst-italic">Followers of</span> <span class="fw-bold">"%(thread name)s"</span>.'
-                  ),
-                  { "thread name": escape(threadName) }
-              )
+            ? sprintf(_t('<span class="fw-bold">To:</span> %(followers)s.'), {
+                  followers: followers.join(", "),
+              })
```

Comment:
I would remove `class="fw-bold"` in the translatable string, and do some processing on string to add this `class="fw-bold"` as styling. That way we can easily change style regardless of translation (and we don't risk of breaking style with translation)

Reply:
Well I think it's ok if the `to:` is directly inside the template?